TradeItIosTicketSDK2/TradeItYahooOAuthCompletionUIFlow.swift

```diff
@@ -54,6 +54,7 @@ class TradeItYahooOAuthCompletionUIFlow: NSObject, TradeItYahooOAuthCompletionVi
                 onNavigationController: navController,
                 asRootViewController: true,
                 showWelcomeScreen: false,
+                hideOpenAccountButton: false,
```

Comment:
No (or not yet ?), but it is using the same linkBrokerUIFlow. I guess I could add a default value for the param.